pom.xml

```diff
@@ -93,6 +90,55 @@ limitations under the License.
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
Why not put in parent?

Reply:
Yes, that would make sense.  I'll prepare a PR for the parent pom.